inference_perf/client/model_servers/README.md

```diff
@@ -0,0 +1,14 @@
+# Model Server Clients  
```

Comment:
I would say, we don't need a separate sub module for model server clients right away. We can start with the model server client being a separate file. As we get more clients, we can split them as needed. Otherwise importing all the submodules requires additional work on the part of the caller.